server/services/auth_service.go

```diff
@@ -100,3 +103,29 @@ func GetUserFromSession(c *fiber.Ctx) (*models.User, error) {
 
 	return user, nil
 }
+func InsertUserToSession(c *fiber.Ctx, user *models.User) error {
+	sess, err := GetStore(c)
+	if err != nil {
+		log.Error(fmt.Printf("this is the error %s", err))
+		return fiber.NewError(fiber.StatusInternalServerError)
```

Comment:
Just return the error, not the status code, the service don't know anything about response or status code, only the controller